docs/examples/Make Test Dataset.ipynb

```diff
@@ -525,7 +525,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.1"
+   "version": "3.9.13"
```

Comment:
please don't downgrade python this much. instead, either undo this change, or make sure you have the latest python 3.11 installed and then rerun the examples using your version (preferably not in this PR though)